### PR TITLE
Inchi

### PR DESCRIPTION
**Pull request by ChayaSt**
## Description
This PR adds RDKit InChI to mol id dictionary. 

## Status
- [ ] Ready to go

### CODE REVIEW

File: cmiles/utils.py [hunk 1]
```diff
@@ -77,44 +89,73 @@ def load_molecule(inp_molecule, backend='openeye'):
...
-        has_openeye = True
-    except ImportError:
-        has_openeye = False
 
+    if backend == 'rdkit':
+        if not has_rdkit:
+            raise RuntimeError("You need to have RDKit installed to use the RDKit backend")
+        molecule = _load_mol_rd(inp_molecule)
+
+    if backend == 'openeye':
```
Comment (dgasmith) [inline cmiles/utils.py]: Probably best handled with a `if/elif/else` here to catch the case if `back = "garbage"`.

File: cmiles/tests/test_cmiles.py [hunk 2]
```diff
@@ -132,27 +133,31 @@ def test_rdkit_isomeric(smiles_input, rd_iso_expected):
         assert cmiles.to_canonical_smiles_rd(rd_mol, isomeric=True, mapped=False, explicit_hydrogen=False) == o
 
 
+@pytest.mark.skipif(rdkit_missing, reason="Cannot test without RDKit")
```
Comment (dgasmith) [inline cmiles/tests/test_cmiles.py]: You can define these like:

```python
using_rdkit = pytest.mark.skipif(rdkit_missing, ...)

@using_rdkit
def test_rdkit...
```

Should save you quite a bit of duplication.

File: cmiles/generator.py [hunk 3]
```diff
@@ -101,9 +110,20 @@ def to_canonical_smiles(molecule, canonicalization='openeye'):
     else:
         raise TypeError("canonicalization must be either 'openeye' or 'rdkit'")
 
+    smiles['Standard_InChI'] = to_inchi(molecule)
```
Comment (dgasmith) [inline cmiles/generator.py]: Great, glad this was easy. Do they have `InChIKey` available as well or is that a straight SHA hash of `InChI`.

Comment (ChayaSt) [reply cmiles/generator.py]: I can get it from RDKit and it's a good idea to do that. RDKit gets it from the InChI API. It's not a straight SHA hash even though it uses SHA hash function for the first 2 blocks. It seems like the `/p` segment is not hashed at all. 
https://www.inchi-trust.org/technical-faq/#13

**Pull request merged**